release/__init__.py

```diff
@@ -64,8 +63,7 @@ def expand_env_vars(config):
 
 
 def load_config(filename):
-    with open(filename) as config_file:
-        return expand_env_vars(yaml.load(config_file))
+    return expand_env_vars(pkgpanda.util.load_yaml(filename))
```

Comment:
The way I see it, when you browse the entire code base, usually traversing multiple files at a time, the consistency in function calls helps while reading the code. Also, usually we don't count the imports as lines of code.